### PR TITLE
Improving Watchman + Other Fixes

### PR DESCRIPTION
**Pull request by imkarthikk**
Improving Watchman with some important fixes:
1. Fixed the Stop Daemon 
2. Fixed Logs and Restart Scripts

@swaathi Can you please verify and close this pull?

### CODE REVIEW

File: watchman_sync [hunk 1]
```diff
@@ -11,14 +11,19 @@
 
 # Change the next 3 lines to suit where you install your script and what you want to call it
 DIR=/usr/local/bin/watchman sync
```
Comment (swaathi) [inline watchman_sync]: Shouldn't this be in `"`?

Comment (swaathi) [inline watchman_sync:13]: Shouldn't this be in `"`?

**Comment by swaathi** [comment]
Can you add one for `watchman_worker` also?

File: watchman_sync (hunk 1 above)
Comment (imkarthikk) [inline watchman_sync:13]: I think this should work. We will have to test it in our Staging server.

**Pull request merged**